infullmvp-kotlin/src/main/java/com/infullmobile/android/infullmvp/InFullMvpView.kt

```diff
@@ -0,0 +1,41 @@
+package com.infullmobile.android.infullmvp
+
+import android.content.Context
+import android.os.Bundle
+import android.util.AttributeSet
+import android.view.View
+import android.widget.FrameLayout
+
+abstract class InFullMvpView<
+        PresenterType : Presenter<PresentedViewType>,
+        out PresentedViewType : PresentedCustomView<PresenterType>>(
+        open val parentContext: Context,
+        attrs: AttributeSet?
+) : FrameLayout(parentContext, attrs) {
+
+    constructor(parentContext: Context) : this(parentContext, null)
+    constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) : this(context, attrs)
+
+    lateinit var view: View
+    abstract val presenter: PresenterType
+    abstract val presentedView: PresentedViewType
+    abstract fun injectIntoGraph()
+
+    override fun onAttachedToWindow() {
+        super.onAttachedToWindow()
+        initialize()
+    }
+
+    fun initialize() {
```

Comment:
should only be used in base test. Maybe "internal" scope would be better.

Reply:
can be only public. They are in a different projects